astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+    "webhookGroupMemberRemoveComplete": 4,
+    "webhookGroupMemberRemoveTransactional": 6,
+    "webhookGroupMemberUpdate": 4,
+    "webhookGroupMemberUpdateComplete": 5,
+    "webhookGroupMemberUpdateTransactional": 6,
+    "webhookGroupUpdate": 4,
+    "webhookGroupUpdateComplete": 4,
+    "webhookGroupUpdateTransactional": 6,
+    "webhookJwtPublicKeyUpdate": 4,
+    "webhookJwtPublicKeyUpdateTransactional": 6,
+    "webhookJwtRefresh": 4,
+    "webhookJwtRefreshTokenRevoke": 4,
+    "webhookJwtRefreshTokenRevokeTransactional": 6,
+    "webhookJwtRefreshTransactional": 6,
+    "webhookKickstartSuccess": 1,
+    "webhookUserAction": 4,
+    "webhookUserBulkCreate": 4,
+    "webhookUserBulkCreateTransactional": 6,
+    "webhookUserCreate": 4,
+    "webhookUserCreateComplete": 4,
+    "webhookUserCreateTransactional": 6,
+    "webhookUserDeactivate": 4,
+    "webhookUserDeactivateTransactional": 6,
+    "webhookUserDelete": 4,
+    "webhookUserDeleteComplete": 5,
+    "webhookUserDeleteTransactional": 6,
+    "webhookUserEmailUpdate": 4,
+    "webhookUserEmailVerified": 4,
+    "webhookUserEmailVerifiedTransactional": 6,
+    "webhookUserIdentityProviderLink": 6,
+    "webhookUserIdentityProviderUnlink": 4,
+    "webhookUserLoginFailed": 4,
+    "webhookUserLoginFailedTransactional": 6,
+    "webhookUserLoginIdDuplicateCreate": 4,
+    "webhookUserLoginIdDuplicateUpdate": 5,
+    "webhookUserLoginNewDevice": 4,
+    "webhookUserLoginNewDeviceTransactional": 6,
+    "webhookUserLoginSuccess": 5,
+    "webhookUserLoginSuccessTransactional": 5,
+    "webhookUserLoginSuspicious": 5,
+    "webhookUserLoginSuspiciousTransactional": 5,
+    "webhookUserPasswordBreach": 4,
+    "webhookUserPasswordBreachTransactional": 6,
+    "webhookUserPasswordResetSend": 4,
+    "webhookUserPasswordResetStart": 4,
+    "webhookUserPasswordResetSuccess": 4,
+    "webhookUserPasswordUpdate": 4,
+    "webhookUserReactivate": 5,
+    "webhookUserReactivateTransactional": 6,
+    "webhookUserRegistrationCreate": 4,
+    "webhookUserRegistrationCreateComplete": 4,
+    "webhookUserRegistrationCreateTransactional": 6,
+    "webhookUserRegistrationDelete": 4,
+    "webhookUserRegistrationDeleteComplete": 5,
+    "webhookUserRegistrationDeleteTransactional": 6,
+    "webhookUserRegistrationUpdate": 4,
+    "webhookUserRegistrationUpdateComplete": 4,
+    "webhookUserRegistrationUpdateTransactional": 6,
+    "webhookUserRegistrationVerified": 4,
+    "webhookUserRegistrationVerifiedTransactional": 6,
+    "webhookUserTwoFactorMethodAdd": 3,
+    "webhookUserTwoFactorMethodRemove": 5,
+    "webhookUserUpdate": 4,
+    "webhookUserUpdateComplete": 4,
+    "webhookUserUpdateTransactional": 6
+  }
+}
+```
+</details>
+
+You can see here that we are collecting information about feature usage.
+
+We will never collect anything personally identifiable. We may collect new metrics as we add new features, but those will be outlined in our release notes.
```

Comment:
> We will never collect anything personally identifiable.

This is not correct. We capture instance Id, which can be tied to our customer. What we promise is that we'll never capture anything personally identifiable about our users users or customers.